modules/infra/src/it/scala/scaladex/infra/GithubClientTests.scala

```diff
@@ -72,8 +72,8 @@ class GithubClientTests extends AsyncFunSpec with Matchers {
       yield issues shouldBe empty
   }
 
-  it("getUserInfo") {
-    for (_ <- client.getUserInfo())
+  it("getUserState") {
+    for (_ <- client.getUserState())
       yield succeed
   }
```

Comment:
@adpi2 

I noticed that this test is now failing on CI.

Previously, `GithubClient#getUserInfo` didn't depend on:

  * `GithubClient#getOrganizationRepositories`
  * `GithubClient#getUserRepositories`  
  *  `GithubClient#getUserOrganizations`

Which seem to be [skipped on CI](https://github.com/scalacenter/scaladex/blob/ce3da52ced67f3fea339ea708f73f3b7414c2489/modules/infra/src/it/scala/scaladex/infra/GithubClientTests.scala#L90-L103).

Does this mean we should also skip this test on CI as well? Sorry for all the questions, I appreciate your help!